### DISCUSSION TITLE
Who owns this organization??  I thought I did...

### DISCUSSION THREAD

**Discussion by SwissFrank** [community/community#29575] [New to GitHub]
A couple months ago I started exploring Github, made a personal account, and found I needed to make an "organization account" in order to have some features (I think, a Wiki).  I tried out one with name XXX, and experimented a bit and then made one with a final name of YYY.  (To the best of my recollection.)

I still have both tabs open on XXX and YYY, but neither shows any members, not even me.  And my menu in the upper right says: Your Profile, Your Repositories, Your Codespaces, etc., but not the "Your Organizations" I was expecting to see, based on

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/accessing-an-organization

So my questions are:

1) organization accounts are always tied to a user account, right?  Many-to-one relationship?

2) when I'm on the organization page, how can I find out what user account it belongs to?  Under "people" I see "We couldn’t find any matching members." 

3) On Settings->Organizations, I see "You are not a member of any organizations."  But I also see a button "Turn MyName into an organization."  I wonder if this is how I made XXX and YYY?  In fact I see in my Chrome password history a saved password for account XXX (which should be a user account, but maybe I turned it into an organization account with this button??

4) How should a 4-person virtual team structure accounts?  Specifically how can I get the organization to pass cleanly to other workers if I make the organization account then get hit by a bus?  Should we make a corporate "personal account" and share its login, or give it to the founder?

Many thanks in advance!

**Comment by queenofcorgis** [comment]
Hey @SwissFrank 👋🏼,

Great questions, I appreciate you asking them! 

1. Organizations can be tied to a personal account OR a business or institution, if you choose the latter the named entity will control the organization account, not the user who set it up.
2. The people page should have the members listed with the owner.
3. That is unlikely given you still have your personal account, for more details check out [this doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/converting-a-user-into-an-organization).
4. GitHub's recommendation is that an organization have at least two people with the owner permissions to ensure no one will lose access to a project, more info [here](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization).

**Reply by SwissFrank** [reply]
Many thanks, royal doggie!

That said, the people page for organizations XXX and YYY are EMPTY: **the "people" tab says "We couldn’t find any matching members."**

And again, the menu in the upper right-hand corner **doesn't even have the "your organizations" menu entry.**

How is this possible?!!?

**Reply by queenofcorgis** [reply]
You're welcome! I've taken a closer look at your account and no organizations are showing on our end. This may be getting into account-specific territory so I recommend opening a ticket on our [Support page](https://support.github.com/contact).

**Comment by SwissFrank** [comment]
Thanks again doggie queen.

It turns out that XXX was a user account I made, to be owner of
organization account YYY.  My account MyName wasn't the owner of YYY, or
any other organization, so naturally I had no organizations and thus no
menu item.  (I've worked in GUIs for 30 years and would suggest instead of
menu items appearing and disappearing like this, or a menu item being
mysteriously unsensitive/grayed out, that the menu item "Your
organizations" always be present and take you to a page that says you have
no organizations.)  Meanwhile, since my user was unrelated to YYY, when
logged in as MyName I could not see the owner XXX.

So, the user confusion was ultimately cleared up.

**Reply by queenofcorgis** [reply]
Thanks for the feedback and letting us know the resolution, glad you were able to clear that up!

**Comment by jfullstackdev** [comment]
> 1. organization accounts are always tied to a user account, right?  Many-to-one relationship?

Orgs from my experience and understanding, is just like a another user but with more granular approaches, so that others can collaborate easily but you can maintain with you as only the owner, 

is it tied ? even the name of the Org itself is not tied, this is one technique to avoid too many users wanting to have many GitHub Pages, for instance,

> 2. when I'm on the organization page, how can I find out what user account it belongs to?  Under "people" I see "We couldn’t find any matching members."

this can be hidden if you are not a member, you can't retrieve it even through API

> 3. On Settings->Organizations, I see "You are not a member of any organizations."  But I also see a button "Turn MyName into an organization."  I wonder if this is how I made XXX and YYY?

you are still a regular user upon checking it 

![image](https://user-images.githubusercontent.com/47092464/185152594-fed163f7-373f-44d0-b70f-bc303b9d950e.png)

> 4. How should a 4-person virtual team structure accounts?  Specifically how can I get the organization to pass cleanly to other workers if I make the organization account then get hit by a bus?  Should we make a corporate "personal account" and share its login, or give it to the founder?

it's highly encouraged to at least have 2 owners, an owner cannot leave the Org without designating another user as owner

then there is now the Successor, in case unfortunate things happened, of course, that should be your decision whether to set it up or not

![image](https://user-images.githubusercontent.com/47092464/185153282-b40c9290-1466-494d-b7fc-e0cab157f9c4.png)
